README.md

```diff
@@ -116,6 +116,28 @@ generates:
             AWSTimestamp: unix_time # gets translated to casual.unix_time
 ```
 
+## Scalar custom value generator
```

Comment:
Can you move this section inside the `scalars` section?
https://github.com/ardeois/graphql-codegen-typescript-mock-data/blob/579c28a9083e15994f23e6b0c5c7867b949289a2/README.md#scalars--scalar-string-scalardefinition--defaultvalue-undefined